src/navigationStore.js

```diff
@@ -471,25 +466,69 @@ function uniteParams(routeName, params) {
   return res;
 }
 
+const defaultSuccess = () => {};
+const defaultFailure = () => {};
+
```

Comment:
My bad here for removing it - they were not being used before.

Reply:
No worries, you just fixed eslint errors. It was used by removed setState, now I re-enabled handlers.